src/containers/ContainerExpressions.js

```diff
@@ -35,17 +35,11 @@ class ContainerExpressionsPanel extends Component {
     addExpression: PropTypes.func,
   };
 
```

Comment:
I think that `componentDidMount` is required here. If the `jwt` exists when this component mounts and `jwt` won't change, you won't be able to fetch expressions.